src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -19,13 +19,43 @@ namespace Cli.Commands
     public class ConfigureOptions : Options
     {
         public ConfigureOptions(
+            string? dataSourceDatabaseType = null,
+            string? dataSourceConnectionString = null,
+            string? dataSourceOptionsDatabase = null,
+            string? dataSourceOptionsContainer = null,
+            string? dataSourceOptionsSchema = null,
+            bool? dataSourceOptionsSetSessionContext = null,
             int? depthLimit = null,
             string? config = null)
             : base(config)
         {
+            DataSourceDatabaseType = dataSourceDatabaseType;
+            DataSourceConnectionString = dataSourceConnectionString;
+            DataSourceOptionsDatabase = dataSourceOptionsDatabase;
+            DataSourceOptionsContainer = dataSourceOptionsContainer;
+            DataSourceOptionsSchema = dataSourceOptionsSchema;
+            DataSourceOptionsSetSessionContext = dataSourceOptionsSetSessionContext;
             DepthLimit = depthLimit;
         }
 
+        [Option("data-source.database-type", Required = false, HelpText = "Database type. Allowed values: MSSQL, PostgreSQL, CosmosDB_NoSQL, MySQL.")]
+        public string? DataSourceDatabaseType { get; }
+
+        [Option("data-source.connection-string", Required = false, HelpText = "Connection string for the data source.")]
+        public string? DataSourceConnectionString { get; }
+
+        [Option("data-source.options.database", Required = false, HelpText = "Database name for Cosmos DB for NoSql.")]
+        public string? DataSourceOptionsDatabase { get; }
+
+        [Option("data-source.options.container", Required = false, HelpText = "Container name for Cosmos DB for NoSql.")]
+        public string? DataSourceOptionsContainer { get; }
+
+        [Option("data-source.options.schema", Required = false, HelpText = "Schema path for Cosmos DB for NoSql.")]
+        public string? DataSourceOptionsSchema { get; }
+
+        [Option("data-source.options.set-session-context", Required = false, HelpText = "Set session context. Allowed values: true, false (default: false).")]
```

Comment:
```suggestion
        [Option("data-source.options.set-session-context", Required = false, HelpText = "Enable session context. Allowed values: true (default), false.")]
```

default is true per datasource.cs line 87